apps/tsilhqotin-language-hub/src/Components/Pages/Teachers/Teachers.tsx

```diff
@@ -1,33 +1,32 @@
-import './Teachers.module.css';
+import ScrollToTop from '../../Widgets/ScrollButton/ScrollToTop';
+import './Teachers.css';
 
 /* eslint-disable-next-line */
 export interface TeachersProps {}
 
 export function Teachers(props: TeachersProps) {
     return (
         <div className="page">
+            <ScrollToTop />
             <div id="heading">
                 <div id="container">
                     <h1 id="pageTitle">Teachers</h1>
                 </div>
             </div>
-            <div className="pageContent">
+            <div id="aboutContent" className="pageContent">
+                <h2>Tŝilhqot’in Alphabet</h2>
+
                 <p>
```

Comment:
I see this page as a one-off for now. It's not worth pulling from a config as it stands.